src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Successful fetched data from database by userId{}", userId);
```

Comment:
The log message in the 'getAllProductsFromShoppingCart' method should be more informative. Include the method name and the user ID in the message to make it clear which method was called and with what parameters. For example: logger.info("getAllProductsFromShoppingCart method was called with userId={}", userId);